x/operator/keeper/usd_value.go

```diff
@@ -299,6 +300,9 @@ func (k *Keeper) CalculateUSDValueForOperator(
 		if isForSlash {
 			// when calculated the USD value for slashing, the input prices map is null
 			// so the price needs to be retrieved here
+			if assetstype.IsNativeToken(assetID) {
+				assetID = strings.Join([]string{assetID, operator}, "_")
```

Comment:
Is this still needed? Seems like the native token can be considered as the base token when fetching price even if different operators own it.

Reply:
removed